PyeonHae/PyeonHae/Network/AuthAPI.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AuthAPI.swift
+//  PyeonHae
+//
+//  Created by 정건호 on 2023/03/25.
+//
+
+import Foundation
+import Alamofire
+
+enum AuthAPI: API {
+    case login([String : String])
+    case logout([String : String])
+    case tokens
+}
+
+extension AuthAPI {
+    
+    var method: HTTPMethod {
+        switch self {
+        case .login, .logout:
+            return .post
+        case .tokens:
+            return .get
+        }
+    }
+    
+    var path: String {
+        switch self {
+        case .login:
+            return "/auth/login"
+        case .logout:
+            return "/auth/logout"
+        case .tokens:
+            return "/auth/tokens"
+        }
+    }
```

Comment:
위에 이 코드를 입력하고

```swift
struct Constants {
    struct AuthAPI {
        static let loginPath = "/auth/login"
        static let logoutPath = "/auth/logout"
        static let tokensPath = "/auth/tokens"
    }
}
```

이 코드를 
```suggestion
var path: String {
    switch self {
    case .login:
        return Constants.AuthAPI.loginPath
    case .logout:
        return Constants.AuthAPI.logoutPath
    case .tokens:
        return Constants.AuthAPI.tokensPath
    }
}
```

이렇게 수정해보는것은 어떨까요?